tests/framework/installer/ceph_installer.go

```diff
@@ -272,8 +272,17 @@ osd_pool_default_size = 1
 	settings := &clusterSettings{h.clusterName, namespace, storeType, dataDirHostPath, mon.Count, 0, usePVC, storageClassName, skipOSDCreation, cephVersion, useCrashPruner}
 	rookCluster := h.Manifests.GetRookCluster(settings)
 	logger.Info(rookCluster)
-	if _, err := h.k8shelper.KubectlWithStdin(rookCluster, createFromStdinArgs...); err != nil {
-		return fmt.Errorf("Failed to create rook cluster : %v ", err)
+	maxTry := 3
+	for i := 0; i < maxTry; i++ {
+		_, err := h.k8shelper.KubectlWithStdin(rookCluster, createFromStdinArgs...)
+		if err == nil {
+			break
+		}
+		if i == maxTry-1 {
+			return fmt.Errorf("Failed to create rook cluster : %v ", err)
+		}
+		logger.Info("Failed to create rook cluster, trying again...")
```

Comment:
```suggestion
		logger.Info("failed to create rook cluster, trying again...")
```

Print the error so we know what failed, that could be useful.

Reply:
```suggestion
		logger.Info("Failed to create rook cluster, trying again... %v", err)
```

Reply:
Done.